faqs/zh/stability-fee.md

```diff
@@ -0,0 +1,145 @@
+# 稳定费 
```

Comment:
There seems to be  significant amount of content missing from this FAQ...

Reply:
I trimmed down and re-arranged some text to be more ELT5 for Chinese audiences, while the wholeness and accuracy are ensured.

Reply:
Were specific questions removed? Why were the detailed examples taken out?

Reply:
No questions were removed and the example is kept. 

However, I compressed the example for the issue I raised [here](https://github.com/makerdao/community/issues/109), and would like to discuss.